packages/eds-core-react/src/components/Select/NativeSelect/NativeSelect.docs.mdx

```diff
@@ -0,0 +1,52 @@
+import { Links, PropsTable, Story } from './../../../../.storybook/components'
+import { NativeSelect } from './..'
+
+# Native Select
+
+Allows users to choose one or multiple items or options from a list.
+
+<Links 
+    ariaUrl="https://www.w3.org/TR/wai-aria-1.1/#select"
+    figmaUrl="https://www.figma.com/file/0bGXR2sCwMVSDNyyzu5BXrO5/UI%E2%80%94User-Interface?node-id=17793%3A67"
+    documentationUrl="https://eds.equinor.com/0b0c666ab/p/31ae8d-select/b/86826a"
+    sourceUrl="https://github.com/equinor/design-system/blob/develop/packages/eds-core-react/src/components/Select/NativeSelect/NativeSelect.tsx"
+    npmUrl="https://www.npmjs.com/package/@equinor/eds-core-react"
+/>
+
+<Story id="inputs-select-nativeselect--introduction" />
+<PropsTable />
+
+## Usage
+
+We recommend using `NativeSelect` on products with limited space as it provides a better user experience.
```

Comment:
"on products with limited space" I'm a little confused what this sentence means. Do you mean in UI's with limited space?